students/piotrowski_stanislaw/lesson_06_dicts_tuples_sets_args_kwargs/capitalize.py

```diff
@@ -0,0 +1,18 @@
+# https://snakify.org/lessons/functions/problems/capitalize/
+# Uppercase
+# piotrsta
+
+
+def capitalize(lower_case_word):
+    first_uppercase_letter_word = []
+    split_list = lower_case_word.split()
+    for elem in split_list:
+        new_elem = chr(ord(elem[0]) - 32) + elem[1:]
+        first_uppercase_letter_word.append(new_elem)
+
+    return print(" ".join(str(elem) for elem in first_uppercase_letter_word))
+
+
+if __name__ == "__main__":
+    input_string = 'harry potter'
+    capitalize(input_string)
```

Comment:
sugestia w treści zadania, była aby funkcja `capitalize` na wejściu przyjęła jedno słowo, czyli zdanie trzeba pociąć przed wywołaniem funkcji  i wywołać dla każdego elementu z listy.
Jest również wbudowana funkcja `title()`, która daje podobny efekt, ale to rozwiązanie ma jedną wadę, ona zamienia na duże litery również znaki po apostrofie:
`he's an engineer, isn't he? -> He'S An Engineer, Isn'T He?`,
a chcielibyśmy, żeby było: `He's An Engineer, Isn't He?`,